contracts/test_argent_account_escape.cairo

```diff
@@ -0,0 +1,113 @@
+use contracts::ArgentAccount;
+use contracts::dummy_syscalls;
+use debug::print_felt;
+
+
+const ESCAPE_SECURITY_PERIOD: felt = 604800; // 7 days
+const ESCAPE_TYPE_GUARDIAN: felt = 1;
+const ESCAPE_TYPE_SIGNER: felt = 2;
+const INITIALIZED_SIGNER: felt = 1;
+const INITIALIZED_GUARDIAN: felt = 2;
+
+#[test]
+#[available_gas(20000)]
+fn valid_trigger_escape_signer() {
+    ArgentAccount::initialize(INITIALIZED_SIGNER, INITIALIZED_GUARDIAN, 0);
+    ArgentAccount::trigger_escape_signer();
+
+    let escape_active_at = ArgentAccount::get_escape_active_at();
+    let escape_type = ArgentAccount::get_escape_type();
+    let block_timestamp = dummy_syscalls::get_block_timestamp();
+    assert(
+        escape_active_at == block_timestamp + ESCAPE_SECURITY_PERIOD,
+        'escape activation set to wrong timestamp'
+    );
+    assert(escape_type == ESCAPE_TYPE_SIGNER, 'wrong escape type');
+}
+
+#[test]
+#[available_gas(20000)]
+#[should_panic(expected = 'argent: cannot override escape')]
+fn trigger_escape_signer_escape_already_active() {
+    ArgentAccount::initialize(INITIALIZED_SIGNER, INITIALIZED_GUARDIAN, 0);
+    ArgentAccount::trigger_escape_signer();
+    ArgentAccount::trigger_escape_signer();
+}
+
+#[test]
+#[available_gas(20000)]
+fn escape_signer() {
+    let new_signer = 5;
+    let future_timestamp = dummy_syscalls::get_block_timestamp()
+        + ESCAPE_SECURITY_PERIOD
+        + 1; // 7 days + current timestamp + 1 block
+    ArgentAccount::initialize(INITIALIZED_SIGNER, INITIALIZED_GUARDIAN, 0);
+    ArgentAccount::trigger_escape_signer();
+    ArgentAccount::escape_signer(new_signer, future_timestamp);
+
+    // check escape cleared
+    let escape_active_at = ArgentAccount::get_escape_active_at();
+    let escape_type = ArgentAccount::get_escape_type();
+    assert(escape_active_at == 0 & escape_type == 0, 'escape not cleared');
+
+    // check new signer
+    let changed_signer = ArgentAccount::get_signer();
+    assert(changed_signer == new_signer, 'signer was not changed to the new one')
+}
+
+#[test]
+#[available_gas(20000)]
+#[should_panic(expected = 'argent: not escaping')]
+fn escape_signer_no_active_escape() {
+    let new_signer = 5;
+    let future_timestamp = dummy_syscalls::get_block_timestamp()
+        + ESCAPE_SECURITY_PERIOD
+        + 1; // 7 days + current timestamp + 1 block
```

Comment:
I find code cleaner without too many linebreaks. Would it work better like this?
```suggestion
    // 7 days + current timestamp + 1 block
    let future_timestamp = dummy_syscalls::get_block_timestamp() + ESCAPE_SECURITY_PERIOD + 1;
```